.tekton/fetch-scanner-data-task.yaml

```diff
@@ -56,9 +56,11 @@ spec:
     script: |
       #!/usr/bin/env bash
       set -euo pipefail
+      dnf -y upgrade --nobest
+      dnf -y install git
       scripts/konflux/fetch-scanner-data.sh "$(params.target-dir)" "$@"
-    # It should not take long to download blobs otherwise there's something odd going on.
-    timeout: 10m
+    # It should not take too long to download blobs otherwise there's something odd going on.
```

Comment:
I suggest to convert this comment into a message (e.g. printed by `fetch-scanner-data.sh`) which would say that the download timeout, if it happens, is likely due to not published blobs and would suggest where to go check publishing status.

Reply:
Updated the message and added to the script.